core/server/models/settings.js

```diff
@@ -219,14 +220,36 @@ Settings = ghostBookshelf.Model.extend({
         });
     },
 
-    populateDefaults: function populateDefaults(unfilteredOptions) {
+    populateDefaults: async function populateDefaults(unfilteredOptions) {
         const options = this.filterOptions(unfilteredOptions, 'populateDefaults');
         const self = this;
 
         if (!options.context) {
             options.context = internalContext.context;
         }
 
+        // this is required for sqlite to pick up the columns after db init
+        await ghostBookshelf.knex.destroy();
+        await ghostBookshelf.knex.initialize();
+
+        // fetch available columns to avoid populating columns not yet created by migrations
+        const columnInfo = await ghostBookshelf.knex.table('settings').columnInfo();
+        const columns = Object.keys(columnInfo);
+
+        // fetch other data that is used when inserting new settings
+        const date = ghostBookshelf.knex.raw('CURRENT_TIMESTAMP');
+        let owner;
+        try {
+            owner = await ghostBookshelf.model('User').getOwnerUser();
+        } catch (e) {
+            // in some tests the owner is deleted and not recreated before setup
```

Comment:
I'm wondering if this is pointing to a deeper issue with db teardown/init in our test setup

Reply:
Yeah - for sure. Not one to solve for this PR though IMO